Automattic/woocommerce-payments
# Fix fatal error for undefined function

#### Changes proposed in this Pull Request
Added a check to avoid the fatal error caused by the function `woocommerce_store_api_register_update_callback` when it does not exist (undefined).

Reported in p1678186111824589-slack-C022WMN88KG  

#### Testing instructions
- Create checkout page with the checkout block.
- Ensure WooPay is enabled in you WCPay settings.
- Go to the shop, add products to the cart and go to the block checkout page.
- Fill in the details, select the checkbox `Save my information for a faster and secure checkout` and provide a phone number.

<img src="https://user-images.githubusercontent.com/33387139/223665018-500faece-0c5e-4a29-afec-e989656514d2.png" width="50%" height="50%" />

- Place order.
- Ensure that your WooPay account has been created.
